Make serve recipe is failing

Seems related to #1087
```
gsurita-30820:kinto gsurita$ make serve 
.venv/bin/kinto --ini config/kinto.ini init
Select the backend you would like to use: (1 - postgresql, 2 - redis, default - memory) 
INFO   Created config /Users/gsurita/kinto/config/kinto.ini
.venv/bin/kinto --ini config/kinto.ini migrate
/Users/gsurita/kinto/kinto/core/initialization.py:567: UserWarning: HTTPS is not enabled
  warnings.warn('HTTPS is not enabled')
INFO   Running kinto 6.0.0.dev0. 
echo '{"name":"kinto","version":"5.3.1-118-g5fc2dedf","source":"https://github.com/gabisurita/kinto/","commit":"5fc2dedfc0a75d9f337ccdbcc935fa051f377164"}' > version.json
.venv/bin/kinto --ini config/kinto.ini start --reload
usage: kinto [-h] [-n NAME] [-s SERVER_TYPE] [--server-name SECTION_NAME]
             [--reload] [--reload-interval RELOAD_INTERVAL] [-b] [-v] [-q]
             [config_uri] [config_vars [config_vars ...]]
kinto: error: unrecognized arguments: http_port=8888
make: *** [serve] Error 2
```

--- a/CHANGELOG.rst
+++ b/CHANGELOG.rst
@@ -29,6 +29,7 @@ Protocol is now at version **1.15**. See `API changelog`_.
 - Remove ``deleted`` field from ``PUT`` requests over tombstones. (#1115)
 - Fix crash when preconditions are used on the permission endpoint (fixes #1066)
 - Fixed resource timestamp upsert in PostgreSQL backend (#1125)
+- Fix pserve argument ordering with Pyramid 1.8 (#1095)
 
 **Internal changes**
 

--- a/CHANGELOG.rst
+++ b/CHANGELOG.rst
@@ -6,6 +6,12 @@ This document describes changes between each past release.
 6.0.0 (unreleased)
 ------------------
 
+**Breaking changes**
+
+- Remove Python 2.7 support and upgrade to Python 3.5. (#1050)
+- Upgraded minimal PostgreSQL support to PostgreSQL 9.5 (#1056)
+- The ``--ini`` parameter is now after the subcommand name (#1095)
+
 **Protocol**
 
 - Fixed ``If-Match`` behavior to match the RFC 2616 specification (#1102).
@@ -13,11 +19,6 @@ This document describes changes between each past release.
   constraint is violated (instead of ``503``) (ref #602)
 
 Protocol is now at version **1.15**. See `API changelog`_.
-
-**Breaking changes**
-
-- Remove Python 2.7 support and upgrade to Python 3.5. (#1050)
-- Upgraded minimal PostgreSQL support to PostgreSQL 9.5 (#1056)
 
 **Bug fixes**
 

--- a/Dockerfile
+++ b/Dockerfile
@@ -13,10 +13,10 @@ RUN \
     apt-get install -y build-essential git python3-dev libssl-dev libffi-dev libpq-dev; \
     pip3 install -e /code[postgresql,monitoring]; \
     pip3 install kinto-pusher kinto-fxa kinto-attachment ; \
-    kinto --ini $KINTO_INI init --host 0.0.0.0 --backend=memory; \
+    kinto init --ini $KINTO_INI --host 0.0.0.0 --backend=memory; \
     apt-get remove -y -qq build-essential git python3-dev libssl-dev libffi-dev libpq-dev; \
     apt-get autoremove -y -qq; \
     apt-get clean -y
 
 # Run database migrations and start the kinto server
-CMD kinto --ini $KINTO_INI migrate && kinto --ini $KINTO_INI start
+CMD kinto migrate --ini $KINTO_INI && kinto start --ini $KINTO_INI

--- a/Makefile
+++ b/Makefile
@@ -86,7 +86,8 @@ version-file:
 	echo '{"name":"$(NAME)","version":"$(VERSION)","source":"$(SOURCE)","commit":"$(COMMIT)"}' > version.json
 
 serve: install-dev $(SERVER_CONFIG) migrate version-file
-	$(VENV)/bin/kinto --ini $(SERVER_CONFIG) start --reload
+# Reload is temporary deactivated because Pylons/pyramid/pull#2962
+	$(VENV)/bin/kinto --ini $(SERVER_CONFIG) start # --reload
 
 migrate: install $(SERVER_CONFIG)
 	$(VENV)/bin/kinto --ini $(SERVER_CONFIG) migrate

--- a/Makefile
+++ b/Makefile
@@ -76,7 +76,7 @@ update-kinto-admin:
 	git add kinto/plugins/admin/build/static/css/main.*
 
 $(SERVER_CONFIG):
-	$(VENV)/bin/kinto --ini $(SERVER_CONFIG) init
+	$(VENV)/bin/kinto init --ini $(SERVER_CONFIG)
 
 NAME := kinto
 SOURCE := $(shell git config remote.origin.url | sed -e 's|git@|https://|g' | sed -e 's|github.com:|github.com/|g')
@@ -87,10 +87,10 @@ version-file:
 
 serve: install-dev $(SERVER_CONFIG) migrate version-file
 # Reload is temporary deactivated because Pylons/pyramid/pull#2962
-	$(VENV)/bin/kinto --ini $(SERVER_CONFIG) start # --reload
+	$(VENV)/bin/kinto start --ini $(SERVER_CONFIG) # --reload
 
 migrate: install $(SERVER_CONFIG)
-	$(VENV)/bin/kinto --ini $(SERVER_CONFIG) migrate
+	$(VENV)/bin/kinto migrate --ini $(SERVER_CONFIG)
 
 tests-once: install-dev version-file install-postgres install-monitoring
 	$(VENV)/bin/py.test --cov-report term-missing --cov-fail-under 100 --cov kinto
@@ -105,7 +105,7 @@ need-kinto-running:
 	@curl http://localhost:8888/v0/ 2>/dev/null 1>&2 || (echo "Run 'make runkinto' before starting tests." && exit 1)
 
 runkinto: install-dev
-	$(VENV)/bin/kinto --ini tests/functional.ini start
+	$(VENV)/bin/kinto start --ini tests/functional.ini
 
 functional: install-dev need-kinto-running
 	$(VENV)/bin/py.test tests/functional.py

--- a/docs/commandline.rst
+++ b/docs/commandline.rst
@@ -10,20 +10,17 @@ and a set of «sub commands» are available.
 
 ::
 
-    usage: kinto [-h] [--ini INI_FILE] [-v]
-                 {init,start,migrate,delete-collection} ...
+    usage: kinto [-h] {init,start,migrate,delete-collection,version} ...
 
     Kinto Command-Line Interface
 
     optional arguments:
       -h, --help            show this help message and exit
-      --ini INI_FILE        Application configuration file
-      -v, --version         Print the Kinto version and exit.
 
     subcommands:
       Main Kinto CLI commands
 
-      {init,start,migrate,delete-collection}
+      {init,start,migrate,delete-collection,version}
                             Choose and run with --help
 
 
@@ -107,7 +104,7 @@ For example:
 
 ::
 
-    kinto --ini=config/postgresql.ini delete-collection --bucket=source --collection=source
+    kinto delete-collection --ini=config/postgresql.ini --bucket=source --collection=source
 
 .. note::
 

--- a/docs/configuration/production.rst
+++ b/docs/configuration/production.rst
@@ -115,7 +115,7 @@ Once a PostgreSQL is up and running somewhere, you should edit your configuratio
 
 .. code-block:: bash
 
-    $ kinto --ini production.ini init
+    $ kinto init --ini production.ini
 
 
 Select your database server address, name and user by editing the configuration file. Also make sure that the PostgreSQL :ref:`backend settings<configuration-backends>` are selected. For our example, the backend configuration would be:
@@ -137,7 +137,7 @@ The last step consists in creating the necessary tables and indices, run the ``m
 
 .. code-block:: bash
 
-    $ kinto --ini production.ini migrate
+    $ kinto migrate --ini production.ini
 
 .. important::
     You should run ``migrate`` every time you change the configuration file or kinto is upgraded.

--- a/docs/core/quickstart.rst
+++ b/docs/core/quickstart.rst
@@ -63,7 +63,7 @@ A generic command is provided to accomplish this:
 
 ::
 
-    $ kinto --ini development.ini migrate
+    $ kinto migrate --ini development.ini
 
 
 Like any *Pyramid* application, it can be served locally with:

--- a/docs/tutorials/custom-id-generator.rst
+++ b/docs/tutorials/custom-id-generator.rst
@@ -60,7 +60,7 @@ And then run your server as usual
 
 .. code-block:: bash
 
-    $ kinto --ini config/kinto.ini start
+    $ kinto start --ini config/kinto.ini
 
 Now, if you try to generate a new record, you should have the new ID used,
 well done!

--- a/kinto/__main__.py
+++ b/kinto/__main__.py
@@ -30,14 +30,6 @@ def main(args=None):
                         required=False,
                         default=DEFAULT_CONFIG_FILE)
 
-    parser.add_argument('-q', '--quiet', action='store_const',
-                        const=logging.CRITICAL, dest='verbosity',
-                        help='Show only critical errors.')
-
-    parser.add_argument('--debug', action='store_const',
-                        const=logging.DEBUG, dest='verbosity',
-                        help='Show all messages, including debug messages.')
-
     commands = ('init', 'start', 'migrate', 'delete-collection', 'version')
     subparsers = parser.add_subparsers(title='subcommands',
                                        description='Main Kinto CLI commands',
@@ -88,6 +80,14 @@ def main(args=None):
                                    help='Listening port number',
                                    required=False,
                                    default=DEFAULT_PORT)
+
+            subparser.add_argument('-q', '--quiet', action='store_const',
+                                   const=logging.CRITICAL, dest='verbosity',
+                                   help='Show only critical errors.')
+
+            subparser.add_argument('-v', '--debug', action='store_const',
+                                   const=logging.DEBUG, dest='verbosity',
+                                   help='Show all messages, including debug messages.')
 
     # Parse command-line arguments
     parsed_args = vars(parser.parse_args(args))

--- a/kinto/__main__.py
+++ b/kinto/__main__.py
@@ -146,12 +146,18 @@ def main(args=None):
 
     elif which_command == 'start':
         pserve_argv = ['pserve']
+
         if parsed_args['reload']:
             pserve_argv.append('--reload')
-        pserve_argv.append('-v')
+
+        if level == logging.DEBUG:
+            pserve_argv.append('-v')
+
+        if level == logging.CRITICAL:
+            pserve_argv.append('-q')
+
         pserve_argv.append(config_file)
         pserve_argv.append('http_port={}'.format(parsed_args['port']))
-        print(pserve_argv)
         pserve.main(argv=pserve_argv)
 
     else:

--- a/kinto/__main__.py
+++ b/kinto/__main__.py
@@ -145,11 +145,14 @@ def main(args=None):
                                          parsed_args['collection'])
 
     elif which_command == 'start':
-        pserve_argv = ['pserve', config_file]
+        pserve_argv = ['pserve']
         if parsed_args['reload']:
             pserve_argv.append('--reload')
+        pserve_argv.append('-v')
+        pserve_argv.append(config_file)
         pserve_argv.append('http_port={}'.format(parsed_args['port']))
-        pserve.main(pserve_argv)
+        print(pserve_argv)
+        pserve.main(argv=pserve_argv)
 
     else:
         print(__version__)

--- a/kinto/__main__.py
+++ b/kinto/__main__.py
@@ -23,13 +23,6 @@ def main(args=None):
 
     parser = argparse.ArgumentParser(description="Kinto Command-Line "
                                                  "Interface")
-    # XXX: deprecate this option, unnatural as first argument.
-    parser.add_argument('--ini',
-                        help='Application configuration file',
-                        dest='ini_file',
-                        required=False,
-                        default=DEFAULT_CONFIG_FILE)
-
     commands = ('init', 'start', 'migrate', 'delete-collection', 'version')
     subparsers = parser.add_subparsers(title='subcommands',
                                        description='Main Kinto CLI commands',
@@ -40,6 +33,20 @@ def main(args=None):
     for command in commands:
         subparser = subparsers.add_parser(command)
         subparser.set_defaults(which=command)
+
+        subparser.add_argument('--ini',
+                               help='Application configuration file',
+                               dest='ini_file',
+                               required=False,
+                               default=DEFAULT_CONFIG_FILE)
+
+        subparser.add_argument('-q', '--quiet', action='store_const',
+                               const=logging.CRITICAL, dest='verbosity',
+                               help='Show only critical errors.')
+
+        subparser.add_argument('-v', '--debug', action='store_const',
+                               const=logging.DEBUG, dest='verbosity',
+                               help='Show all messages, including debug messages.')
 
         if command == 'init':
             subparser.add_argument('--backend',
@@ -80,14 +87,6 @@ def main(args=None):
                                    help='Listening port number',
                                    required=False,
                                    default=DEFAULT_PORT)
-
-            subparser.add_argument('-q', '--quiet', action='store_const',
-                                   const=logging.CRITICAL, dest='verbosity',
-                                   help='Show only critical errors.')
-
-            subparser.add_argument('-v', '--debug', action='store_const',
-                                   const=logging.DEBUG, dest='verbosity',
-                                   help='Show all messages, including debug messages.')
 
     # Parse command-line arguments
     parsed_args = vars(parser.parse_args(args))

--- a/tests/test_main.py
+++ b/tests/test_main.py
@@ -143,7 +143,7 @@ class TestMain(unittest.TestCase):
             res = main(['--ini', TEMP_KINTO_INI, 'start', '--reload'])
             assert res == 0
             assert mocked_pserve.call_count == 1
-            assert '--reload' in mocked_pserve.call_args[0][0]
+            assert '--reload' in mocked_pserve.call_args[1]['argv']
 
     def test_cli_can_display_kinto_version(self):
         with mock.patch('sys.stdout', new_callable=StringIO) as mock_stdout:

--- a/tests/test_main.py
+++ b/tests/test_main.py
@@ -23,7 +23,7 @@ class TestMain(unittest.TestCase):
             pass
 
     def test_cli_init_generates_configuration(self):
-        res = main(['--ini', TEMP_KINTO_INI, 'init', '--backend', 'memory'])
+        res = main(['init', '--ini', TEMP_KINTO_INI, '--backend', 'memory'])
         assert res == 0
         assert os.path.exists(TEMP_KINTO_INI)
 
@@ -31,24 +31,22 @@ class TestMain(unittest.TestCase):
         with open(TEMP_KINTO_INI, 'w') as f:
             f.write("exists")
         with mock.patch('sys.stderr', new_callable=StringIO) as mock_stderr:
-            res = main(['--ini', TEMP_KINTO_INI, 'init',
-                        '--backend', 'memory'])
+            res = main(['init', '--ini', TEMP_KINTO_INI, '--backend', 'memory'])
             assert res == 1
             content = mock_stderr.getvalue()
             assert '{} already exists.'.format(TEMP_KINTO_INI) in content
 
     def test_cli_init_asks_for_backend_if_not_specified(self):
         with mock.patch("kinto.__main__.input", create=True, return_value="2"):
-            res = main(['--ini', TEMP_KINTO_INI, 'init'])
+            res = main(['init', '--ini', TEMP_KINTO_INI])
             assert res == 0
         with open(TEMP_KINTO_INI) as f:
             content = f.read()
         assert 'redis' in content
 
     def test_cli_init_asks_until_backend_is_valid(self):
-        with mock.patch("kinto.__main__.input", create=True,
-                        side_effect=["10", "2"]):
-            res = main(['--ini', TEMP_KINTO_INI, 'init'])
+        with mock.patch("kinto.__main__.input", create=True, side_effect=["10", "2"]):
+            res = main(['init', '--ini', TEMP_KINTO_INI])
             assert res == 0
             with open(TEMP_KINTO_INI) as f:
                 content = f.read()
@@ -57,8 +55,8 @@ class TestMain(unittest.TestCase):
     def test_fails_if_not_enough_args(self):
         with mock.patch('sys.stderr', new_callable=StringIO) as mock_stderr:
             with pytest.raises(SystemExit) as excinfo:
-                main(['--ini', TEMP_KINTO_INI])
-            assert 'INI_FILE' in mock_stderr.getvalue()
+                main([])
+            assert 'arguments are required: subcommand' in mock_stderr.getvalue()
             assert excinfo.value.code == 2
 
     def test_cli_init_installs_postgresql_dependencies_if_needed(self):
@@ -72,9 +70,8 @@ class TestMain(unittest.TestCase):
 
         with mock.patch('builtins.__import__', side_effect=psycopg2_missing):
             with mock.patch('pip.main', return_value=None) as mocked_pip:
-                with mock.patch("kinto.__main__.input", create=True,
-                                return_value="1"):
-                    res = main(['--ini', TEMP_KINTO_INI, 'init'])
+                with mock.patch("kinto.__main__.input", create=True, return_value="1"):
+                    res = main(['init', '--ini', TEMP_KINTO_INI])
                     assert res == 0
                     assert mocked_pip.call_count == 1
 
@@ -89,15 +86,13 @@ class TestMain(unittest.TestCase):
 
         with mock.patch('builtins.__import__', side_effect=redis_missing):
             with mock.patch('pip.main', return_value=None) as mocked_pip:
-                with mock.patch("kinto.__main__.input", create=True,
-                                return_value="2"):
-                    res = main(['--ini', TEMP_KINTO_INI, 'init'])
+                with mock.patch("kinto.__main__.input", create=True, return_value="2"):
+                    res = main(['init', '--ini', TEMP_KINTO_INI])
                     assert res == 0
                     assert mocked_pip.call_count == 1
 
     def test_main_takes_sys_argv_by_default(self):
-        testargs = ["prog", "--ini", TEMP_KINTO_INI, 'init',
-                    '--backend', 'memory']
+        testargs = ['prog', 'init', '--ini', TEMP_KINTO_INI, '--backend', 'memory']
         with mock.patch.object(sys, 'argv', testargs):
             main()
 
@@ -107,20 +102,18 @@ class TestMain(unittest.TestCase):
 
     def test_cli_migrate_command_runs_init_schema(self):
         with mock.patch('kinto.__main__.scripts.migrate') as mocked_migrate:
-            res = main(['--ini', TEMP_KINTO_INI, 'init',
-                        '--backend', 'memory'])
+            res = main(['init', '--ini', TEMP_KINTO_INI, '--backend', 'memory'])
             assert res == 0
-            res = main(['--ini', TEMP_KINTO_INI, 'migrate'])
+            res = main(['migrate', '--ini', TEMP_KINTO_INI])
             assert res == 0
             assert mocked_migrate.call_count == 1
 
     def test_cli_delete_collection_run_delete_collection_script(self):
         with mock.patch('kinto.__main__.scripts.delete_collection') as del_col:
             del_col.return_value = mock.sentinel.del_col_code
-            res = main(['--ini', TEMP_KINTO_INI, 'init',
-                        '--backend', 'memory'])
+            res = main(['init', '--ini', TEMP_KINTO_INI, '--backend', 'memory'])
             assert res == 0
-            res = main(['--ini', TEMP_KINTO_INI, 'delete-collection',
+            res = main(['delete-collection', '--ini', TEMP_KINTO_INI,
                         '--bucket', 'test_bucket',
                         '--collection', 'test_collection'])
             assert res == mock.sentinel.del_col_code
@@ -128,39 +121,35 @@ class TestMain(unittest.TestCase):
 
     def test_cli_start_runs_pserve(self):
         with mock.patch('kinto.__main__.pserve.main') as mocked_pserve:
-            res = main(['--ini', TEMP_KINTO_INI, 'init',
-                        '--backend', 'memory'])
+            res = main(['init', '--ini', TEMP_KINTO_INI, '--backend', 'memory'])
             assert res == 0
-            res = main(['--ini', TEMP_KINTO_INI, 'start'])
+            res = main(['start', '--ini', TEMP_KINTO_INI])
             assert res == 0
             assert mocked_pserve.call_count == 1
 
     def test_cli_start_with_quiet_option_runs_pserve_with_quiet(self):
         with mock.patch('kinto.__main__.pserve.main') as mocked_pserve:
-            res = main(['--ini', TEMP_KINTO_INI, 'init',
-                        '--backend', 'memory'])
+            res = main(['init', '--ini', TEMP_KINTO_INI, '--backend', 'memory'])
             assert res == 0
-            res = main(['--ini', TEMP_KINTO_INI, 'start', '-q'])
+            res = main(['start', '-q', '--ini', TEMP_KINTO_INI])
             assert res == 0
             assert mocked_pserve.call_count == 1
-            assert mocked_pserve.call_args_list[0][1]['argv'][1] == '-q'
+            assert mocked_pserve.call_args[1]['argv'][1] == '-q'
 
     def test_cli_start_with_verbose_option_runs_pserve_with_verbose(self):
         with mock.patch('kinto.__main__.pserve.main') as mocked_pserve:
-            res = main(['--ini', TEMP_KINTO_INI, 'init',
-                        '--backend', 'memory'])
+            res = main(['init', '--ini', TEMP_KINTO_INI, '--backend', 'memory'])
             assert res == 0
-            res = main(['--ini', TEMP_KINTO_INI, 'start', '-v'])
+            res = main(['start', '-v', '--ini', TEMP_KINTO_INI])
             assert res == 0
             assert mocked_pserve.call_count == 1
-            assert mocked_pserve.call_args_list[0][1]['argv'][1] == '-v'
+            assert mocked_pserve.call_args[1]['argv'][1] == '-v'
 
     def test_cli_start_with_reload_runs_pserve_with_reload(self):
         with mock.patch('kinto.__main__.pserve.main') as mocked_pserve:
-            res = main(['--ini', TEMP_KINTO_INI, 'init',
-                        '--backend', 'memory'])
+            res = main(['init', '--ini', TEMP_KINTO_INI, '--backend', 'memory'])
             assert res == 0
-            res = main(['--ini', TEMP_KINTO_INI, 'start', '--reload'])
+            res = main(['start', '--reload', '--ini', TEMP_KINTO_INI])
             assert res == 0
             assert mocked_pserve.call_count == 1
             assert '--reload' in mocked_pserve.call_args[1]['argv']
@@ -173,24 +162,21 @@ class TestMain(unittest.TestCase):
 
     def test_cli_can_configure_logger_in_quiet(self):
         with mock.patch('kinto.__main__.logging') as mocked_logging:
-            main(['-q', '--ini', TEMP_KINTO_INI, 'init',
-                        '--backend', 'memory'])
+            main(['init', '-q', '--ini', TEMP_KINTO_INI, '--backend', 'memory'])
             mocked_logging.basicConfig.assert_called_with(
                 level=mocked_logging.CRITICAL,
                 format=DEFAULT_LOG_FORMAT)
 
     def test_cli_can_configure_logger_in_debug(self):
         with mock.patch('kinto.__main__.logging') as mocked_logging:
-            main(['--debug', '--ini', TEMP_KINTO_INI, 'init',
-                  '--backend', 'memory'])
+            main(['init', '--debug', '--ini', TEMP_KINTO_INI, '--backend', 'memory'])
             mocked_logging.basicConfig.assert_called_with(
                 level=mocked_logging.DEBUG,
                 format=DEFAULT_LOG_FORMAT)
 
     def test_cli_use_default_logging_logger(self):
         with mock.patch('kinto.__main__.logging') as mocked_logging:
-            main(['--ini', TEMP_KINTO_INI, 'init',
-                  '--backend', 'memory'])
+            main(['init', '--ini', TEMP_KINTO_INI, '--backend', 'memory'])
             mocked_logging.basicConfig.assert_called_with(
                 level=logging.INFO,
                 format=DEFAULT_LOG_FORMAT)

--- a/tests/test_main.py
+++ b/tests/test_main.py
@@ -135,6 +135,26 @@ class TestMain(unittest.TestCase):
             assert res == 0
             assert mocked_pserve.call_count == 1
 
+    def test_cli_start_with_quiet_option_runs_pserve_with_quiet(self):
+        with mock.patch('kinto.__main__.pserve.main') as mocked_pserve:
+            res = main(['--ini', TEMP_KINTO_INI, 'init',
+                        '--backend', 'memory'])
+            assert res == 0
+            res = main(['--ini', TEMP_KINTO_INI, 'start', '-q'])
+            assert res == 0
+            assert mocked_pserve.call_count == 1
+            assert mocked_pserve.call_args_list[0][1]['argv'][1] == '-q'
+
+    def test_cli_start_with_verbose_option_runs_pserve_with_verbose(self):
+        with mock.patch('kinto.__main__.pserve.main') as mocked_pserve:
+            res = main(['--ini', TEMP_KINTO_INI, 'init',
+                        '--backend', 'memory'])
+            assert res == 0
+            res = main(['--ini', TEMP_KINTO_INI, 'start', '-v'])
+            assert res == 0
+            assert mocked_pserve.call_count == 1
+            assert mocked_pserve.call_args_list[0][1]['argv'][1] == '-v'
+
     def test_cli_start_with_reload_runs_pserve_with_reload(self):
         with mock.patch('kinto.__main__.pserve.main') as mocked_pserve:
             res = main(['--ini', TEMP_KINTO_INI, 'init',

